install.sh

```diff
@@ -707,6 +710,10 @@ pushd .
     tensile_opt="${tensile_opt} -DTensile_LIBRARY_FORMAT=yaml"
   fi
 
+  if [[ "${asm_debug}" == true ]]; then
```

Comment:
That checking build_release == false should be enough.
I didn't see the necessary to add asm_debug here unless you want to add another argument for install.sh to control the TENSILE_ASM_DEBUG directly.

Reply:
Yea, you're right, will change soon.